player/js/elements/htmlElements/HShapeElement.js

```diff
@@ -182,10 +182,26 @@ HShapeElement.prototype.calculateBoundingBox = function (itemsData, boundingBox)
       this.calculateShapeBoundingBox(itemsData[i], boundingBox);
     } else if (itemsData[i] && itemsData[i].it) {
       this.calculateBoundingBox(itemsData[i].it, boundingBox);
+    } else if (itemsData[i] && itemsData[i].style && itemsData[i].w ) {
+      this.expandStrokeBoundingBox(itemsData[i].w, boundingBox);
     }
   }
 };
 
+HShapeElement.prototype.expandStrokeBoundingBox = function (widthProperty, boundingBox) {
+  var width = 0;
+  if (widthProperty.keyframes) {
+    width = Math.max(...widthProperty.keyframes.map(kf => (kf.s ?? 0) * widthProperty.mult));
```

Comment:
for performance reasons, I'm not transpiling the code to ES5, so I try to stick with the ES5 available features. Would you mind chainging this to a for loop and not using the Null coalescing operator?